docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -13,7 +13,7 @@ redirect_from:
 {% capture overview %}
 
 This quickstart shows you how to easily install a Kubernetes cluster on machines
-running Ubuntu 16.04, CentOS 7 or HypriotOS v1.0.1+. The installation uses a
+running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+. The installation uses a
```

Comment:
Nothing significant changes in the other versions... I've happily run kubeadm in all three (16.04, 16.10 and 17.04)